docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Setup the base image
+FROM python:3.8-slim-bullseye
+
+# Activate venv
+RUN python3.8 -m venv /venv
+ENV PATH="venv/bin:$PATH"
+
+# Setup DeepSparse
+RUN pip3 install deepsparse[server]
+
```

Comment:
We should make bash the default entrypoint. Partially to encourage CLI commands as the first point of use and partially because when python is the entrypoint, exiting to bash is non-intuitive, if possible.

```suggestion

# Bash as default entrypoint
CMD ["bin/bash"]

```